lib/terraforming/resource/iam_policy_attachment.rb

```diff
@@ -48,11 +48,29 @@ def attachment_name_from(policy)
       end
 
       def entities_for_policy(policy)
-        @client.list_entities_for_policy(policy_arn: policy.arn)
+        # list_entities_for_policy is a weird one: the response class
+        # has three different member variables that we need to
+        # paginate through altogether.
+        result = Aws::IAM::Types::ListEntitiesForPolicyResponse.new
+        result.policy_groups = []
+        result.policy_users = []
+        result.policy_roles = []
+        
+        fetch_proc = Proc.new { |marker| @client.list_entities_for_policy(policy_arn: policy.arn, marker: marker) }
+        data_proc = Proc.new { |resp| [resp] }
+        paginate_results(fetch_proc, data_proc).each do |resp|
+          require 'pp'
+          STDERR.puts resp.pretty_inspect
```

Comment:
These two lines seems to be inserted for debugging.
Please make sure that commit does not contain any debugging code :octocat: